evals/configs/femnist/conf.yml

```diff
@@ -9,9 +9,8 @@ ps_ip: 10.0.0.1
 # E.g., master node has 4 available processes, then 1 for the ps, and worker should be set to: worker:3
 worker_ips: 
     - 10.0.0.1:[2]
-    - 10.0.0.2:[2]
 
-exp_path: $HOME/FedScale/fedscale/core
+exp_path: $FEDSCALE_HOME/FedScale/fedscale/core
```

Comment:
$FEDSCALE_HOME includes FedScale; this and the rest of the similar things should start with $FEDSCALE_HOME/fedscale